client/packages/system/src/Patient/api/operations.graphql

```diff
@@ -261,3 +264,37 @@ query insurances(
     }
   }
 }
+
+mutation insertInsurance($storeId: String!, $input: InsertInsuranceInput!) {
+  insertInsurance(storeId: $storeId, input: $input) {
+    ... on IdResponse {
+      id
+    }
+  }
+}
+
+mutation updateInsurance($storeId: String!, $input: UpdateInsuranceInput!) {
+  updateInsurance(storeId: $storeId, input: $input) {
+    ... on IdResponse {
+      id
+    }
+  }
+}
+
```

Comment:
Just noticed the GraqphQL mutation input doesn't quite match up with the database. The "policy_number_person" and "policy_number_family" fields are nullable, so they shouldn't be required, right?